Documentation/ApiOverview/SiteHandling/SiteSettingDefinitions.rst

```diff
@@ -79,6 +79,10 @@ Site setting definition properties
         ..  confval:: description
             :type: string
             :name: site-settings-definition-settings-description
+            :Example: 'Configure `baz` to be used in `bar`.'
+
+            The description can make use of markdown syntax for rich text
+            formatting.
```

Comment:
I prefer an additional note about quoting the text in single quotes when using spaces and/or Mark-Down specific characters

```suggestion
            While Markdown syntax can be used in YAML to provide rich text formatting, there are
            a few gotchas. Because YAML is sensitive to special characters and indentation, you
            might need to wrap your Markdown text in single quotes (') to prevent it from breaking
            the YAML syntax.
```